shared/styles/style-guide-colors.js

```diff
@@ -7,6 +7,8 @@ export default {
   blue3: '#a8d7ff',
   blue4: '#e6f3ff',
 
+  blue3_40: 'rgba(168, 215, 255, 0.4)',
```

Comment:
ok spoke to @cecileboucheron . we're going to rename the colors to use _* as the opacity. so blue3_40 is correct but we should update the rest so like black40 -> black_40 so we can be consistent